source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -249,6 +252,17 @@ message FactIdEquation {
 
 }
 
+message FactIdIsIrrelevant {
+
+  // Records a fact that |result_id| is irrelevant (i.e. it's usage doesn't
+  // change the semantics of the module). This implies that a use of this id
+  // can later be replaced with some other id of the same type.
```

Comment:
"... some other id of the same type, or the definition of |result_id| can be changed so that it yields a different value."

Reply:
Fixed in 9a66484.